dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/process_definition.py

```diff
@@ -21,13 +21,15 @@
 from datetime import datetime
 from typing import Any, Dict, List, Optional, Set
 
+
 from pydolphinscheduler import configuration
 from pydolphinscheduler.constants import TaskType
 from pydolphinscheduler.core.resource import Resource
 from pydolphinscheduler.exceptions import PyDSParamException, PyDSTaskNoFoundException
 from pydolphinscheduler.java_gateway import JavaGate
 from pydolphinscheduler.models import Base, Project, Tenant, User
 from pydolphinscheduler.utils.date import MAX_DATETIME, conv_from_str, conv_to_schedule
+from pydolphinscheduler.resources_plugin.__init__ import ResourcePlugin
```

Comment:
Not sure it work or not, but if we can import ResourcePlugin from module `resources_plugin` will be better
```suggestion
from pydolphinscheduler.resources_plugin import ResourcePlugin
```